src/main/docker/gitlab-gitlabci.yml

```diff
@@ -0,0 +1,52 @@
+version: "3.6"
+services:
+    gitlab:
+        build: gitlab
+        container_name: gitlab
```

Comment:
"http://gitlab" would work without naming the containers, why is that needed ? :)

Reply:
Yes, for accessing the container from the other container we do not need it.
But later it is necessary to execute a command (`sudo docker exec -it gitlab grep 'Password:' /etc/gitlab/initial_root_password`) in the container. There we have to specify the container name. If I were to remove it, the container name would be something like `docker_gitlab_1`, which is just as functional as the current solution, but just adds extra work to the input.

Reply:
Ok :)